### DISCUSSION TITLE
Can we get the trace of a reverted transaction?

### DISCUSSION THREAD

**Discussion by Billy1900** [remix-run/remix#6878] [Q&A]
I have a function which I plan to revert, since the REVERT instruction provides a way to stop execution and revert state changes, without consuming all provided gas and with the ability to return a reason. At the same time, I want to get the trace of the transaction, can we do that using geth? I have tried to use hardhat/truffle/ganachi, but it does not work (it seems like the framework refuses to send the reverted transaction, but we can forcefully do it on remix IDE).

```
function attack1_int_overflow() public {
      multiVulnToken.transferProxy(
          address(this),
          attacker1Address,
          uint256(2 ** 255 + 1),
          uint256(2 ** 255)
      );
      revert();
  }
```

**Comment by kiliman** [comment]
I believe you want the Remix IDE project from Ethereum. This is the Remix Web Framework.

https://github.com/ethereum/remix-project
